### PR TITLE
ci: fail checkstyle on all findings except warning.LOW

### PR DESCRIPTION
**Pull request by jdrueckert**
Currently, the analysis stage only fails for findings of severity "error".
Findings of severities "warning.HIGH", "warning.NORMAL" and "warning.LOW" do not result in a failure.
This PR attempts to reduce the acceptable severity to "warning.LOW" only, all findings of higher severities should fail the build.

### CODE REVIEW

**Comment by jdrueckert** [comment]
Meh, that somehow does not work as expected :thinking:

**Comment by pollend** [comment]
I think I fixed the syntax errors @jdrueckert

File: facades/PC/src/main/java/org/terasology/engine/Terasology.java [hunk 1]
```diff
@@ -346,8 +346,9 @@ private static int getLastNumber(String str) {
      * <p>
      * On Windows, you may be able to set a limit using one of these external tools:
      * <ul>
-     *     <li><a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/devtest/application-verifier">Application Verifier
-     *         (<code>AppVerif.exe</code>)</a>, available from the Windows SDK</li>
+     *     <li>
+     *     <a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/devtest/application-verifier">Application Verifier (<code>AppVerif.exe</code>)</a>,
```
Comment (pollend) [inline facades/PC/src/main/java/org/terasology/engine/Terasology.java:350]: any clue about getting rid of this LineLenghtCheck. sometimes its better to format this with one long line. 

I think this is a lot more readable then trying to work out how to chop the line to the right length. 
```
     * <ul>
     *     <li><a href="https://docs.microsoft.com/en-us/windows-hardware/drivers/devtest/application-verifier">Application Verifier (<code>AppVerif.exe</code>)</a>,available from the Windows SDK</li>
     *     <li><a href="https://github.com/lowleveldesign/process-governor">Process Governor (<code>procgov</code>)</a>,an open source third-party tool</li>
     * </ul>
```

**Comment by pollend** [comment]
maybe we can adjust the checkstyle warnings: https://checkstyle.sourceforge.io/property_types.html#SeverityLevel

File: facades/PC/src/main/java/org/terasology/engine/Terasology.java (hunk 1 above)
Comment (jdrueckert) [reply facades/PC/src/main/java/org/terasology/engine/Terasology.java]: As @skaldarnar mentioned on Discord, for these "sometimes" cases there's annotations that allow exactly that: ignoring the check rule in a particular case.

**Comment by jdrueckert** [comment]
[Make all but "low" Checkstyle findings fail the build](https://trello.com/c/eOcmr9id/208-make-all-but-low-checkstyle-findings-fail-the-build)

**Review by pollend — APPROVED** [review]

**Pull request merged**